src/Our.Umbraco.DocTypeGridEditor/Extensions/ContentTypeServiceExtensions.cs

```diff
@@ -9,12 +9,15 @@ internal static class ContentTypeServiceExtensions
     {
         public static string GetAliasByGuid(this IContentTypeService contentTypeService, Guid id)
         {
-            return ApplicationContext.Current.ApplicationCache.RuntimeCache.GetCacheItem<string>(
+            var cache = Current.AppCaches.RequestCache;
+
+            return cache.GetCacheItem<string>(
                 string.Concat("Our.Umbraco.DocTypeGridEditor.Web.Extensions.ContentTypeServiceExtensions.GetAliasById_", id),
-                () => ApplicationContext.Current
-                    .DatabaseContext
-                    .Database
-                    .ExecuteScalar<string>("SELECT [cmsContentType].[alias] FROM [cmsContentType] INNER JOIN [umbracoNode] ON [cmsContentType].[nodeId] = [umbracoNode].[id] WHERE [umbracoNode].[uniqueID] = @0", id));
+                () =>
+                {
+                    using (var scope = Current.ScopeProvider.CreateScope())
+                        return scope.Database.ExecuteScalar<string>("SELECT [cmsContentType].[alias] FROM [cmsContentType] INNER JOIN [umbracoNode] ON [cmsContentType].[nodeId] = [umbracoNode].[id] WHERE [umbracoNode].[uniqueID] = @0", id);
+                });
```

Comment:
@skttl I think this need to call `scope.Complete();` or add `autoComplete: true` in `CreateScope()`.
https://our.umbraco.com/forum/umbraco-8/96075-alternative-solution-to-use-applicationcontextcurrentdatabasecontextdatabase-in-umbraco-8